test_runner/fixtures/compare_fixtures.py

```diff
@@ -0,0 +1,172 @@
+import pytest
+from contextlib import contextmanager
+from abc import ABC, abstractmethod
+
+from fixtures.zenith_fixtures import PgBin, PgProtocol, VanillaPostgres, ZenithEnv
+from fixtures.benchmark_fixture import MetricReport, ZenithBenchmarker
+
+# Type-related stuff
+from typing import Iterator
+
+
+class PgCompare(ABC):
```

Comment:
A bigger comment block would be nice, explaining what PgCompare is for, how to use it, etc. Either here, or at the top of the file.

Reply:
Oh, I see there's a comment like that in `zenith_with_baseline`. Maybe that's enough, or maybe that should be moved/copied to the top of the file.

Reply:
expanded the zenith_with_baseline docstring with intent/instructions, and added pointer to it from the PgCompare docstring.